src/server/databaseHelper.py

```diff
@@ -1,8 +1,9 @@
 import sqlite3
+import databaseClasses
```

Comment:
If you replace this with `import * from databaseClasses` you won't have to prefix all of them with `databaseClasses.xxxx`. If you want to keep the prefixing, but think the module name is too long, you can do `import databaseClasses as dbcls` and then use `dbcls.xxxx` instead. Up to you, any of the 3 is fine.

Reply:
I made the changes to the import, not sure if it's right since black is giving me an error when it tries to format it

Reply:
What's the error?

Reply:
error: cannot format E:\CS-471-blue-team\src\server\databaseHelper.py: Cannot parse: 2:7: import * from databaseClasses

Reply:
Ah, sorry. It's `from databaseClasses import *`. Remember to change `databaseClasses.xxx` to just `xxx` afterwards.

Reply:
yup already did that, I'll push the fixed import statement